.github/workflows/publish.yml

```diff
@@ -106,6 +107,12 @@ jobs:
         run: |
           cargo build --package nargo_cli --release --target ${{ matrix.target }} --no-default-features --features plonk_bn254_wasm
 
+      - uses: actions/cache/save@v3
+        if: ${{ github.event_name != 'merge_group' }} # Don't create cache entries for the merge queue.
+        with:
+          path: ${{ steps.restore-cache.inputs.path }}
```

Comment:
This doesn't seem allowed. I'm seeing this error in the run:
> Input required and not supplied: path

Reply:
Perhaps we needs to make the inputs env vars?

Reply:
argh, that's kinda yucky but no choice I guess.